examples/fluids/qfunctions/shocktube.h

```diff
@@ -0,0 +1,527 @@
+// Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-734707. All Rights
+// reserved. See files LICENSE and NOTICE for details.
+//
+// This file is part of CEED, a collection of benchmarks, miniapps, software
+// libraries and APIs for efficient high-order finite element and spectral
+// element discretizations for exascale applications. For more information and
+// source code availability see http://github.com/ceed.
+//
+// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
+// a collaborative effort of two U.S. Department of Energy organizations (Office
+// of Science and the National Nuclear Security Administration) responsible for
+// the planning and preparation of a capable exascale ecosystem, including
+// software, applications, hardware, advanced system engineering and early
+// testbed platforms, in support of the nation's exascale computing imperative.
+
+/// @file
+/// Shock tube initial condition and Euler equation operator for Navier-Stokes
+/// example using PETSc - modified from eulervortex.h
+
+// Model from:
+//   On the Order of Accuracy and Numerical Performance of Two Classes of
+//   Finite Volume WENO Schemes, Zhang, Zhang, and Shu (2011).
+
+#ifndef shocktube_h
+#define shocktube_h
+
+#ifndef __CUDACC__
+#  include <math.h>
+#endif
```

Comment:
It's safe to just include these days.
```suggestion
#include <math.h>
```